### PR TITLE
7316 update ra tocreate data artifacts

### PR DESCRIPTION
**Pull request by kellykelly3**

### CODE REVIEW

File: RecentActivity/src/org/sleuthkit/autopsy/recentactivity/Chromium.java [hunk 1]
```diff
@@ -243,7 +244,7 @@ private void getHistory(String browser, String browserLocation) {
             }
             List<HashMap<String, Object>> tempList;
             tempList = this.dbConnect(temps, HISTORY_QUERY);
-            logger.log(Level.INFO, "{0}- Now getting history from {1} with {2}artifacts identified.", new Object[]{moduleName, temps, tempList.size()}); //NON-NLS
+            logger.log(Level.INFO, "{0}- Now getting history from {1} with {2}artifacts identified.", new Object[]{getName(), temps, tempList.size()}); //NON-NLS
```
Comment (markmckinnon) [inline RecentActivity/src/org/sleuthkit/autopsy/recentactivity/Chromium.java:247]: Space between {2}artifacts

File: RecentActivity/src/org/sleuthkit/autopsy/recentactivity/Chromium.java [hunk 2]
```diff
@@ -696,7 +687,7 @@ private void getLogins(String browser, String browserLocation) {
                 break;
             }
             List<HashMap<String, Object>> tempList = this.dbConnect(temps, LOGIN_QUERY);
-            logger.log(Level.INFO, "{0}- Now getting login information from {1} with {2}artifacts identified.", new Object[]{moduleName, temps, tempList.size()}); //NON-NLS
+            logger.log(Level.INFO, "{0}- Now getting login information from {1} with {2}artifacts identified.", new Object[]{getName(), temps, tempList.size()}); //NON-NLS
```
Comment (markmckinnon) [inline RecentActivity/src/org/sleuthkit/autopsy/recentactivity/Chromium.java:690]: Space between {2}artifacts

File: RecentActivity/src/org/sleuthkit/autopsy/recentactivity/Chromium.java [hunk 3]
```diff
@@ -838,7 +830,7 @@ private void getAutofill(String browser, String browserLocation) {
                 : AUTOFILL_QUERY;
 
         List<HashMap<String, Object>> autofills = this.dbConnect(dbFilePath, autoFillquery);
-        logger.log(Level.INFO, "{0}- Now getting Autofill information from {1} with {2}artifacts identified.", new Object[]{moduleName, dbFilePath, autofills.size()}); //NON-NLS
+        logger.log(Level.INFO, "{0}- Now getting Autofill information from {1} with {2}artifacts identified.", new Object[]{getName(), dbFilePath, autofills.size()}); //NON-NLS
```
Comment (markmckinnon) [inline RecentActivity/src/org/sleuthkit/autopsy/recentactivity/Chromium.java:833]: Space between {2}artifacts

File: RecentActivity/src/org/sleuthkit/autopsy/recentactivity/Chromium.java [hunk 4]
```diff
@@ -906,7 +899,7 @@ private void getFormAddressArtifacts(AbstractFile webDataFile, String dbFilePath
 
         // Get Web form addresses
         List<HashMap<String, Object>> addresses = this.dbConnect(dbFilePath, webformAddressQuery);
-        logger.log(Level.INFO, "{0}- Now getting Web form addresses from {1} with {2}artifacts identified.", new Object[]{moduleName, dbFilePath, addresses.size()}); //NON-NLS
+        logger.log(Level.INFO, "{0}- Now getting Web form addresses from {1} with {2}artifacts identified.", new Object[]{getName(), dbFilePath, addresses.size()}); //NON-NLS
```
Comment (markmckinnon) [inline RecentActivity/src/org/sleuthkit/autopsy/recentactivity/Chromium.java:902]: Space between {2}artifacts

File: RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractIE.java [hunk 5]
```diff
@@ -558,33 +562,31 @@ private boolean executePasco(String indexFilePath, String outputFileName) {
...
-                    bbattributes.add(new BlackboardAttribute(ATTRIBUTE_TYPE.TSK_DOMAIN,
-                            RecentActivityExtracterModuleFactory.getModuleName(), domain));
-                }
-                bbattributes.add(new BlackboardAttribute(ATTRIBUTE_TYPE.TSK_USER_NAME,
-                        RecentActivityExtracterModuleFactory.getModuleName(), user));
-                bbart.addAttributes(bbattributes);
+            Collection<BlackboardAttribute> bbattributes = new ArrayList<>();
+            bbattributes.add(new BlackboardAttribute(ATTRIBUTE_TYPE.TSK_URL,
+                    RecentActivityExtracterModuleFactory.getModuleName(), realurl));
+            //bbattributes.add(new BlackboardAttribute(ATTRIBUTE_TYPE.TSK_URL_DECODED.getTypeID(), "RecentActivity", EscapeUtil.decodeURL(realurl)));
```
Comment (markmckinnon) [inline RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractIE.java:568]: Should the commented code stay?

File: RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractIE.java [hunk 6]
```diff
@@ -558,33 +562,31 @@ private boolean executePasco(String indexFilePath, String outputFileName) {
...
+
+            bbattributes.add(new BlackboardAttribute(ATTRIBUTE_TYPE.TSK_DATETIME_ACCESSED,
+                    RecentActivityExtracterModuleFactory.getModuleName(), ftime));
+            bbattributes.add(new BlackboardAttribute(ATTRIBUTE_TYPE.TSK_REFERRER,
+                    RecentActivityExtracterModuleFactory.getModuleName(), ""));
+            // @@@ NOte that other browser modules are adding TITLE in hre for the title
```
Comment (markmckinnon) [inline RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractIE.java:574]: I know it is not yours but can you correct hre

File: RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java [hunk 7]
```diff
@@ -783,7 +771,8 @@ private boolean parseAutopsyPluginOutput(String regFilePath, AbstractFile regFil
                                         Long usbMtime = Long.parseLong(artnode.getAttribute("mtime")); //NON-NLS
                                         usbMtime = Long.valueOf(usbMtime.toString());
 
-                                        BlackboardArtifact bbart = regFile.newArtifact(ARTIFACT_TYPE.TSK_DEVICE_ATTACHED);
+//                                        BlackboardArtifact bbart = regFile.newArtifact(ARTIFACT_TYPE.TSK_DEVICE_ATTACHED);
```
Comment (markmckinnon) [inline RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java:774]: Should this commented out code stay here?

File: RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java [hunk 8]
```diff
@@ -800,11 +789,11 @@ private boolean parseAutopsyPluginOutput(String regFilePath, AbstractFile regFil
                                         bbattributes.add(new BlackboardAttribute(ATTRIBUTE_TYPE.TSK_DEVICE_MAKE, parentModuleName, make));
                                         bbattributes.add(new BlackboardAttribute(ATTRIBUTE_TYPE.TSK_DEVICE_MODEL, parentModuleName, model));
                                         bbattributes.add(new BlackboardAttribute(ATTRIBUTE_TYPE.TSK_DEVICE_ID, parentModuleName, value));
-                                        bbart.addAttributes(bbattributes);
+//                                        bbart.addAttributes(bbattributes);
```
Comment (markmckinnon) [inline RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractRegistry.java:792]: Should this commented out code stay here?

File: RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractIE.java (hunk 5 above)
Comment (kellykelly3) [reply RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractIE.java:568]: @markmckinnon that line was previously commented out for reason I don't know.  My feeling is that I should leave it?

Comment (markmckinnon) [reply RecentActivity/src/org/sleuthkit/autopsy/recentactivity/ExtractIE.java:568]: I agree then.

**Review by markmckinnon — APPROVED** [review]

**Pull request merged**